barretenberg/cpp/src/barretenberg/vm/avm/trace/public_inputs.hpp

```diff
@@ -46,6 +47,7 @@ inline void read(uint8_t const*& it, GasSettings& gas_settings)
     read(it, gas_settings.gas_limits);
     read(it, gas_settings.teardown_gas_limits);
     read(it, gas_settings.max_fees_per_gas);
+    read(it, gas_settings.max_priority_fees_per_gas);
```

Comment:
I think we also need to update fee calculation in `AvmTraceBuilder::pay_fee`

Reply:
Oh right! Didn't realise fee computation is already implemented there.